es6/README.md

```diff
@@ -51,6 +51,56 @@ globals:
 The boolean value indicates whether the global is to be treated as
 read-only (`false`) or read-write (`true`).
 
+## Naming
+
+Classes and constructor functions should be TitleCased
+
+```javascript
+// GOOD
+class DatePicker { }
+
+var LinkLabel = function() { }
+
+// BAD
+class linkLabel { }
+```
+
+Constant values that appear outside of a method scope should be in ALL_CAPS
+
+```javascript
+// GOOD
+export const DEFAULT_NAME = 'Element'
+
+const STATE_CALLBACKS = {
+    pending: sendPendingResponse,
+    completed: sendCompletedResponse,
+    archived: sendArchivedResponse
+}
+
+// BAD
+export const defaultName = 'Element'
+
+class DatePicker {
+    render() {
+        // WRONG: constant is within a method
+        const DEFAULT_VALUE = 0
+    }
+}
+```
+
+All other symbols (methods, variables, constants within a method) should be camelCased!
+
+```javascript
+// GOOD
+class MyClass {
+    const name = 'Button'
+    render() { ... }
+}
+
+// BAD - `const` outside of a method should be ALL_CAPS
+const defaultName = 'Element'
```

Comment:
As I'm re-reading this, we don't follow this in the `platform-scaffold`.  We export many const symbols that are not `SCREAMING_CAPS`, but they're usually functions. I wonder if the distinction of when to use `SCREAMING_CAPS` is that the value is "data" (ie. it's not a function):

```js
// GOOD - exporting a function
export const receiveProductData = createAction('Receive Product Data')

// BAD - exporting _data_
export const defaultName = 'Element'
```